packages/build-essentials/src/styles/styleConstants.js

```diff
@@ -26,7 +26,7 @@ const config = {
         fullScreenClose: ['context'],
         drawer: ['context'],
         bar: ['context'],
-        primaryToolbar: ['context'],
+        primaryToolbar: '2',
```

Comment:
We have old views (like the image media select) and there the z-index of the close button is 3. So we need a minimum value of 4.